main.js

```diff
@@ -22,18 +32,57 @@ async function main() {
                 branchesToDelete.push(pull.data.head.ref)
             }
         }
-
+        
+        if (prefix) {
+            const branchFunc = await client.paginate("GET /repos/{owner}/{repo}/branches", {
```

Comment:
Does octokit not have any standard function for getting branches? Do we need to insert the endpoint here?

Reply:
I could not identify a function on octokit that operates like this. That was my first attempt but no luck